core/trino-main/src/main/java/io/trino/sql/planner/LogicalPlanner.java

```diff
@@ -121,7 +121,7 @@
 import static io.trino.SystemSessionProperties.isCollectPlanStatisticsForAllQueries;
 import static io.trino.metadata.MetadataUtil.createQualifiedObjectName;
 import static io.trino.spi.StandardErrorCode.CATALOG_NOT_FOUND;
-import static io.trino.spi.StandardErrorCode.INVALID_ARGUMENTS;
+import static io.trino.spi.StandardErrorCode.CONSTRAINT_VIOLATION;
```

Comment:
Please update the commit message:

> Use CONSTRAINT_VIOLATION for NOT NULL violations

And remove the description

Reply:
Done.